### DISCUSSION TITLE
The search results are case sensitive

### DISCUSSION THREAD

**Discussion by vigneshvdm** [microsoft-search/pnp-modern-search#1530] [Q&A]
We are using the modern search in one of our site collections which has a lot of documents

When we search for iron and IRON the results are different

**To Reproduce**
1. Go to the search page
2. Search for some text 
3.  open the search page in new tab
4. search for the same text, but this time in ALL CAPS
5. Compare the results and the results shown will be different

**Expected behavior**
Case insensitive search results. It should return the same results irrespective of the casing of the words

I tried the same variation in out of the box search and it is returning the same set of results for both iron and IRON

http://company.sharepoint.com/sites/sitename/_layouts/15/search.aspx/siteall?q=iron

**Desktop (please complete the following information):**
 - Google Chrome and Edge

**Comment by wobba** [comment]
Search is case insensitive, except for KQL keywords. All mentioned at https://docs.microsoft.com/en-us/sharepoint/dev/general-development/keyword-query-language-kql-syntax-reference

Thus iron and IRON should yield the exact same results regarding of using OOB search or the web parts. Also, depending on where you search OOB, results are cached.

**Reply by vigneshvdm** [reply]
Thank you @wobba 

Yes that is what we expect as well, for the search to be case insensitive. 

But when we search for IRON and iron in the PNP Modern search, the order and the results are different in search results.

Not sure if there is a setting in PNP search results webpart to make it case insensitive. 

Regards,
Vicky

**Reply by wobba** [reply]
The API is case insensitive. Nothing we can control one way or the other. Can you try the same using the SharePoint Search Query Tool? https://github.com/pnp/PnP-Tools/releases/tag/v2.8.8-Search-QueryTool
